deploy/resources.go

```diff
@@ -147,9 +147,9 @@ var assets = func() http.FileSystem {
f\xe1\x39\xe8\x12\xb4\x69\x00\x61\xd0\x24\xc7\xd0\x72\xc1\xe4\x08\xf1\x5f\xdf\xfd\xd6\x83\xf5\x3e\xdf\x40\xea\x04\x1f\xe0\x5d\xc8\x57\xa4\x23\x5e\xbd\x89\x60\xce\x9a\xb2\xd5\x5e\x3c\xd0\x5e\xf1\xca\x38\xec\xe6\xb0\xd1\x6a\x4b\x94\xaf\xc4\x3d\x65\x3c\x19\xc2\x06\x95\x1a\x55\xee\x72\x23\x78\xc2\x5d\x89\x30\xf4\xd8\x72\x61\xfd\x23\xda\x5b\xcd\x6e\xe6\x37\xef\x6f\x26\x01\x3b\x52\xaf\x94\x3b\xd4\xda\x78\x58\x4a\x4a\x54\x85\x4e\xaa\x51\x3c\xe9\x66\x8b\x5f\xae\x2e\x57\x04\x65\xf2\x06\xe2\x95\xd0\x29\x56\x59\xda\xb2\xf0\x85\xc5\xe8\xd5\x53\x2d\xfc\x70\xb8\xd2\xbc\x5a\x06\x2d\x87\x8e\xe5\x2f\x1a\x57\x7c\x11\x89\x7d\x49\xf6\x3e\x89\xd7\x0d\xdd\xef\x25\x71\x5d\x07\x03\xa2\x32\x31\xb1\x23\x02\xa9\x60\x74\x63\x73\x8f\xf6\x5e\xe2\x66\xbc\x31\x76\x2d\x75\x3a\x22\x65\x1d\x05\x8d\x70\x63\x4e\xb5\xc7\x43\xfe\xf3\x2c\x8a\x78\x86\x7a\x3a\x59\xbc\xfc\xcf\xa0\x8d\xf6\x71\xe3\x27\x93\x56\xc5\xd6\x2f\x89\x7c\xaf\x66\x55\x0f\xf7\xe0\x6d\x32\x97\xcd\x4a\xc6\xab\x6a\xe4\x5a\x7a\xe2\xee\xea\xc4\x41\x26\x92\xe0\xc2\x85\xde\x7e\x73\xf5\x26\xc6\x16\x96\x70\xda\x8e\xca\x43\x0f\x23\xa1\x13\xfa\xdf\x49\xe7\xe9\xfe\x93\x39\x59\xc8\x13\x0d\xfb\xf3\xf4\xfd\x9f\xa3\xf4\x85\x7c\xb2\x15\xf7\xe6\x76\x85\x95\x8f\xe6\x76\x9f\xef\xa6\x9c\xdb\x09\xc5\x93\x70\x2f\xef\x51\x6d\x9b\x99\x5e\x39\x17\x64\x36\xbc\xe6\x31\x4a\x7b\x11\x5b\x4f\x7d\xf1\x21\x57\x32\x96\x5e\xb5\x15\x85\x3d\xe4\x74\xe7\xfb\xac\xba\xfd\x19\x7f\xc8\x3b\xcb\x9c\xdf\x79\x61\x79\xee\x13\x10\xea\x4e\xfb\x8f\xd3\xfd\x50\x06\xbc\xa4\xfd\x2f\x69\x7f\xdb\x82\x97\xb4\xff\x25\xed\x7f\x49\xfb\x5f\xd2\xfe\x97\xb4\xff\x25\xed\x7f\x49\xfb\x5f\xd2\xfe\x97\xb4\xff\x9b\xa5\xfd\xed\xe7\x29\xc3\x21\xe8\x41\x07\x51\x07\x27\x2a\x79\xf1\x69\x67\x2a\x63\xa3\xc3\x28\xe8\x28\xf7\x3c\x3c\x86\x58\x2e\x3b\x9e\xaa\x74\x0c\xf0\x7a\x10\xac\xa1\xd5\x93\xb4\xba\x7c\xf1\x21\x49\xac\x0d\xb8\x6d\x78\xe5\x29\x75\x42\xcb\x49\x1f\x8b\xe0\x38\x1d\xeb\x4f\xa9\x95\x70\x7e\x6e\x85\x76\xb2\xfa\x46\xe0\x84\xe1\xd7\x95\x70\x1e\xbc\xcc\xb0\xca\x88\x4a\x12\x7c\x0d\x08\x93\x70\x14\xd8\x68\x2c\x05\xd6\xa5\xe3\x06\x84\x36\x7e\x85\xb6\x3d\x91\xac\x0e\x85\x25\xc2\xe3\xa8\x73\xa2\xfc\x88\xa5\x10\x91\x9f\xf9\x44\xf1\x89\x04\xce\xf9\xa4\xf8\x8e\x48\x4a\x9a\x6a\x2a\x37\xc2\xd5\xe7\x93\xbf\x1d\xce\x19\x3a\xd7\x79\xbe\xe0\x60\x14\xb9\x2a\x32\xa1\xc1\xa2\x48\xf8\x73\xa0\xf2\x55\x4a\x06\x65\x2c\xb8\x08\x4e\xd0\x0b\xa9\x1c\x88\x85\x29\xba\x26\xb0\x24\xcb\x9d\x04\xdb\x49\x7b\x04\x69\x8b\xc2\x75\x05\xa8\x23\x06\x87\xc5\xf5\x59\xc3\x9a\xc1\xaf\x5c\xc9\xfb\xe7\xe1\x72\xec\x27\x3a\x70\x29\x7d\xc4\x2e\xbf\x0f\x68\x9c\x87\xef\xe1\x96\x30\xb7\x05\x9e\xc3\x47\xa1\x1c\x9e\xc3\x67\xbd\xd6\x66\xf3\x34\x8c\xba\xc7\xee\xc7\x23\x77\xb3\xac\xbe\xc3\xa8\x3e\xd6\xa8\x11\x7b\xc2\xde\xdd\x23\xdc\x51\x97\x79\x76\x9c\x41\x7e\x74\x56\x7b\x3c\x7f\xe7\x8f\x7b\x7a\x5d\x2a\x7f\xee\x73\xec\x4d\x3b\x48\x6a\x41\xe1\xe0\xd6\xee\x23\xa8\xb7\x42\xe5\x2b\xf1\x76\x77\xaf\xfc\x5a\x29\x7c\x66\xd4\x78\x1c\x8e\xbb\x61\x32\x01\x6f\xcb\xe3\xd5\xce\x1b\x4b\x16\x18\xee\xfc\x3f\x00\x00\xff\xff\xd9\xac\x86\xa4\xa4\x39\x00\x00"),
```

Comment:
It doesn't seem like it is related to that PR.

Reply:
This was updated after running make. I've seen this happening a few times. Was this file out of date in master?

Reply:
Yes, I think it has already been updated in master by the time you branched it off.

Reply:
I just rebased and that update went away! :)

Reply:
👍🏼